backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -476,4 +476,9 @@ public String getOrderTestDate() {
     // order_test_date = test_date for antigen testing
     return getTestDate();
   }
+
+  @JsonProperty("Site_of_care")
+  public String getSiteOfCare() {
+    return testEvent.getOrganization().getOrganizationType();
```

Comment:
we are also getting the organization and passing its name to report stream via the `Organization_name` property. I think we should follow the pattern of the optional class properties and initializing them in the constructor.

Currently `Organization_name` uses `testEvent.facility.organization.name` and this uses `testEvent.organization.type`

Reply:
👍